### ISSUE TITLE
Any shortcut to acces RecentItems?

### ISSUE THREAD

**Issue by mortenskoett** [bananenfisch/RecentItems#1]
Hi!

Great extension! Thanks

I was wondering if there is any kind of keyboard shortcut to acces the plugin so that I don't have to use the touchpad?

Thanks!

**Comment by bananenfisch** [comment]
Good idea - goiing to do this. But so busy now :-( ... hope until summer.
Devs are welcome to help me with this simple small but usefull extension... :-)

**Comment by molitona** [comment]
@bananenfisch 

pls fix it for gnome 45
